scripts/start-colima.py

```diff
@@ -36,13 +36,15 @@ def main(argv: Sequence[str] | None = None) -> int:
     HOME = os.path.expanduser("~")
     rc = subprocess.call(
         (
-            f"{HOME}/.local/share/sentry-devenv/bin/colima",
+            "colima",
```

Comment:
this is fixed now with the `pathprepend=f"{reporoot}/.devenv/bin",` to `sentry devservices up` in sync.py